### PR TITLE
Move TiUP docs to TiDB Tools and add entry for TiDB Operator

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

1. Rename **TiDB Ecosystem Tools** to **TiDB Tools** to avoid confusion of external ecosystem tools.
2. Move TiUP docs to **TiDB Tools**. 
3. In TiDB Tools, add an entry for TiDB Operator. 
4. Add TiUP and TiDB Operator to TiDB Tools overview, user case, and download documents. 

The changes for TiUP are only applicable to 4.0 and later releases, so this PR needs modification after being applied to other branches. 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: download-ecosystem-tools.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
 
 本页面汇总了 TiDB 周边工具官方维护版本的下载链接。
 
+## TiUP
+
+TiUP 安装过程十分简洁，无论是 Darwin 还是 Linux 操作系统，执行一行命令即可安装成功。详情请参考[安装 TiUP](/tiup/tiup-overview.md#安装-tiup)。
+
+## TiDB Operator
+
+TiDB Operator 运行在 Kubernetes 集群。在搭建好 Kubernetes 集群并创建了 Custom Resource Definition (CRD) 后，你可以选择在线部署或者离线安装 TiDB Operator。详情请参考 [在 Kubernetes 上部署 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-tidb-operator)。
```
Comment (DanielZhangQD) [inline download-ecosystem-tools.md:16]:
```suggestion
TiDB Operator 运行在 Kubernetes 集群。在搭建好 Kubernetes 集群后，你可以选择在线或者离线部署 TiDB Operator。详情请参考 [在 Kubernetes 上部署 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-tidb-operator)。
```

File: ecosystem-tool-user-guide.md [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+- 适用 TiDB 版本：v4.0 及以上
+
+### 在 Kubernetes 上部署运维 TiDB - TiDB Operator
+
+[TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable) 是 Kubernetes 上的 TiDB 集群自动运维系统，提供包括部署、升级、扩缩容、备份恢复、配置变更的 TiDB 全生命周期管理。借助 TiDB Operator，TiDB 可以无缝运行在公有云或私有部署的 Kubernetes 集群上。
+
+基本信息：
+
+- [TiDB Operator 架构](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/architecture)
+- [在 Kubernetes 上部署运维 TiDB 快速上手](/tiup/tiup-component-management.md)
```
Comment (DanielZhangQD) [inline ecosystem-tool-user-guide.md:31]: TiUP

Comment (qiancai) [reply ecosystem-tool-user-guide.md:31]:
```suggestion
- [在 Kubernetes 上部署运维 TiDB 快速上手](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/get-started/)
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: tidb-operator.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: TiDB Operator
+summary: 了解 TiDB Operator
```
Comment (lilin90) [inline tidb-operator.md:3]: `summary` 一般用完整的一句话，句末加句号。
```suggestion
summary: 了解 Kubernetes 上的 TiDB 集群自动部署运维工具 TiDB Operator。
```

File: TOC.md [hunk 4]
```diff
@@ -160,11 +160,27 @@
...
+    - [故障排查](/tiup/tiup-troubleshooting-guide.md)
+    - [参考手册](/tiup/tiup-reference.md)
+    - [TiDB 集群拓扑文件配置](/tiup/tiup-cluster-topology-reference.md)
+    - [TiUP 镜像参考指南](/tiup/tiup-mirror-reference.md)
+    - TiUP 组件文档
+      - [tiup-playground 运行本地测试集群](/tiup/tiup-playground.md)
+      - [tiup-cluster 部署运维生产集群](/tiup/tiup-cluster.md)
+      - [tiup-mirror 定制离线镜像](/tiup/tiup-mirror.md)
+      - [tiup-bench 进行 TPCC/TPCH 压力测试](/tiup/tiup-bench.md)
+  - [TiDB Operator](tidb-operator.md)
```
Comment (lilin90) [inline TOC.md:183]:
```suggestion
  - [TiDB Operator](/tidb-operator.md)
```

File: ecosystem-tool-user-case.md [hunk 5]
```diff
@@ -1,13 +1,21 @@
...
+summary: 本文档介绍 TiDB 工具的常见适用场景与工具选择。
 aliases: ['/docs-cn/dev/ecosystem-tool-user-case/']
 ---
 
-# TiDB 生态工具适用场景
+# TiDB 工具适用场景
 
 本文档从生态工具的适用场景出发，介绍部分常见场景下的生态工具选择。
 
+## 在本地环境部署运维 TiDB
```
Comment (qqqdan) [inline ecosystem-tool-user-case.md:11]: `在物理机/虚拟机上部署运维 TiDB - TiUP`

**Review by qqqdan — APPROVED** [review]

File: TOC.md [hunk 6]
```diff
@@ -160,10 +160,26 @@
...
+    - [故障排查](/tiup/tiup-troubleshooting-guide.md)
+    - [参考手册](/tiup/tiup-reference.md)
+    - [TiDB 集群拓扑文件配置](/tiup/tiup-cluster-topology-reference.md)
+    - [TiUP 镜像参考指南](/tiup/tiup-mirror-reference.md)
+    - TiUP 组件文档
+      - [tiup-playground 运行本地测试集群](/tiup/tiup-playground.md)
+      - [tiup-cluster 部署运维生产集群](/tiup/tiup-cluster.md)
+      - [tiup-mirror 定制离线镜像](/tiup/tiup-mirror.md)
+      - [tiup-bench 进行 TPCC/TPCH 压力测试](/tiup/tiup-bench.md)
+  - [TiDB Operator](tidb-operator-overview.md)
```
Comment (lilin90) [inline TOC.md:182]:
```suggestion
  - [TiDB Operator](/tidb-operator-overview.md)
```

File: tidb-operator-overview.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: TiDB Operator
+summary: 了解 TiDB Operator
```
Comment (lilin90) [inline tidb-operator-overview.md:3]:
```suggestion
summary: 了解 Kubernetes 上的 TiDB 集群自动部署运维工具 TiDB Operator。
```

File: ecosystem-tool-user-guide.md [hunk 8]
```diff
@@ -1,13 +1,44 @@
...
-title: TiDB 生态工具功能概览
+title: TiDB 工具功能概览
 aliases: ['/docs-cn/dev/ecosystem-tool-user-guide/','/docs-cn/dev/reference/tools/user-guide/','/docs-cn/dev/how-to/migrate/from-mysql/', '/docs-cn/dev/how-to/migrate/incrementally-from-mysql/', '/docs-cn/dev/how-to/migrate/overview/', '/docs-cn/dev/reference/tools/use-guide/']
 ---
 
-# TiDB 生态工具功能概览
+# TiDB 工具功能概览
 
-TiDB 提供了丰富的生态工具，可以帮助你进行数据迁移和校验，例如：数据迁入、全量导出、全量导入、备份恢复、校验等等。请根据需要选择适用的工具。
+TiDB 提供了丰富的工具，可以帮助你进行部署运维、数据管理（例如，数据迁移、备份恢复、数据校验）、在 TiKV 上运行 Spark SQL。请根据需要选择适用的工具。
```
Comment (leoppro) [inline ecosystem-tool-user-guide.md:8]: DM 从这一页删掉后，还要提数据迁移、备份恢复吗？

**Review by leoppro — APPROVED** [review]
LGTM

File: ecosystem-tool-user-guide.md (hunk 8 above)
Comment (qiancai) [reply ecosystem-tool-user-guide.md:8]: DM 还在这页，没有删掉

**Comment by allengaoo** [comment]
LGTM

**Review by lilin90 — COMMENTED** [review]
LGTM

**Comment by sunzhaoyang** [comment]
LGTM

**Review by lilin90 — APPROVED** [review]

**Pull request merged**